src/queue/QueueItem.ts

```diff
@@ -0,0 +1,13 @@
+import { Message, VoiceChannel } from 'discord.js';
+
+export default class QueueItem {
+  public sound: string;
```

Comment:
Mark all of these as `readonly`. Always prefer immutable classes where possible! Note that readonly != immutable but it's better than nothing 😁